arrays/arrays.c

```diff
@@ -21,30 +21,37 @@ typedef struct Array {
  *****/
 Array *create_array (int capacity) {
   // Allocate memory for the Array struct
-
+  Array *array = malloc(sizeof(Array));
   // Set initial values for capacity and count
-
+  array->capacity = cap;
```

Comment:
I'm not sure what "cap" is referring to here. The passed in variable is "capacity" I would use that.